dev-lang/yab/yab-1.7.8.recipe

```diff
@@ -0,0 +1,55 @@
+SUMMARY="Extended version of the yabasic programming language"
+DESCRIPTION="yab is an extended version of yabasic, a BASIC programming \
+language, with special commands designed for Haiku. "
+HOMEPAGE="http://software.besly.de/"
+COPYRIGHT="1995-2006 Marc-Oliver Ihm (yabasic)
+	2006-2009 Jan Bungeroth (yab improvements)
```

Comment:
This should be:
 2006-2009 Jan Bungeroth (yab)

as yab was Jan's idea.

Reply:
I'll try to see if I get around to it today to fix this, thanks for the pointer! :+1: 
Or maybe @lorglas or you @bbjimmy could file a new PR to fix this :)

Reply:
Fixed in https://github.com/haikuports/haikuports/pull/4913